pkg/visor/visorconfig/parse.go

```diff
@@ -24,11 +25,37 @@ func Parse(log *logging.Logger, r io.Reader, confPath string, visorBuildInfo *bu
 	if err != nil {
 		return nil, compat, err
 	}
-	log.WithField("config version: ", conf.Version).Info()
-
+	if conf.Version != "unknown" {
+		log.WithField("config version: ", conf.Version).Info()
+	}
 	// we check if the version of the visor and config are the same
 	if (conf.Version != "unknown") && (visorBuildInfo.Version != "unknown") {
-		compat = strings.Contains(strings.Split(visorBuildInfo.Version, "-")[0], strings.Split(conf.Version, "-")[0])
+		v1, err := semver.Make(strings.TrimPrefix(conf.Version, "v"))
```

Comment:
We can name this `cVer` and the one below `vVer`.

Reply:
changed